python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb

```diff
@@ -307,101 +309,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import whylogs as why\n",
     "from datetime import datetime, timezone\n",
     "current_date = datetime.now(timezone.utc)\n",
-    "profile = why.log(df).profile()\n",
-    "profile.set_dataset_timestamp(current_date)"
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "We're also setting the profile's dataset timestamp as the current datetime. If this is not set, the Writer would simply assign the current datetime automatically to the profile."
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## ✍️ The WhyLabs Writer"
+    "profile = why.log(df, dataset_timestamp=current_date, name=\"tour\")"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, you can simply create a WhyLabsWriter object and use it to send your profiles.\n",
-    "The process is very similar to sending a regular profile. The only difference is you should the `option` method in order to pass `reference_profile_name` to indicate and name this profile as a static Reference Profile:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from whylogs.api.writer.whylabs import WhyLabsWriter\n",
-    "\n",
-    "writer = WhyLabsWriter().option(reference_profile_name=\"my_reference_profile\")\n",
-    "writer.write(file=profile.view())"
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "> Another way of setting `reference_profile_name` is to define the environment variable `WHYLABS_REFERENCE_PROFILE_NAME`\n",
```

Comment:
@jamie256 Is this true?? I can't find any reference to that name/feature.

Reply:
https://github.com/search?q=repo%3Awhylabs%2Fwhylogs+get_whylabs_refernce_profile_name&type=code

The environment variable reference profile name functionality still exists.

Reply:
lol I think I added that for feature parity, but I had no idea if it existed before. It's fine I guess, it's there now.